sdk/servicebus/service-bus/CHANGELOG.md

```diff
@@ -6,6 +6,12 @@
   [PR 9233](https://github.com/Azure/azure-sdk-for-js/pull/9233)
   [PR 9284](https://github.com/Azure/azure-sdk-for-js/pull/9284)
 
+### Breaking Changes
```

Comment:
Can you merge #9703 and update this PR? I think you are using a different changelog format there

Reply:
Yup